### ISSUE TITLE
Move dimension after placement

### ISSUE THREAD

**Issue by horsetailfiddlehead** [hamish2014/FreeCAD_drawing_dimensioning#71]
Definitely not a bug, but it might be really handy to be have the ability to move/translate a dimension after it has been placed, just as can be done with the dimension text. Nothing really fancy, you wouldn't be redefining the dimension, simply moving the location of the dimensioning lines, etc back and forth. (hopefully that explanation is clear enough).

I Often find myself needing to move a dimension farther or closer to a part as I start adding other dimensions to ensure everything remains readable. It is cumbersome to delete a dimension, then redo the dimension, only to have to repeat that again sometimes if the location isn't good.

**Comment by hamish2014** [comment]
Agreed having to recreate dimensions to redefine them is cumbersome.
